html-aam/index.html

```diff
@@ -580,7 +580,10 @@ <h4 id="el-area-no-href">`area` <span class="el-context">(no `href` attribute)</
             </tr>
             <tr>
               <th>Comments</th>
-              <td></td>
+              <td>
+                User agents MAY still expose an `a` element lacking a `href` attribute with a `link` role in the event an author specifies interactive behavior for the element. E.g., by using an
```

Comment:
`E.g., by using an...`

Small nit: this seems like an incomplete sentence. If it's a continuation of the previous sentence, would a comma or semicolon be better here? Such as:

User agents MAY still expose an `a` element lacking a `href` attribute with a `link` role in the event an author specifies interactive behavior for the element, e.g., using an...

Reply:
```suggestion
                User agents MAY still expose an `a` element lacking the `href` attribute with a `link` role in the event an author specifies interactive behavior for the element. For example, if using an
```